src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Link } from './Link'
+
+export default {
+  title: 'Link',
+  parameters: {
+    info: `Link component
+
+    Based on Typography guide: https://designsystem.digital.gov/components/typography/#links`,
+  },
+}
+
+export const Default = (): React.ReactElement => (
+  <p>
+    <Link href={'#'}> This</Link> is a text link.
+  </p>
+)
+
+/*
+ * TODO: sort out how to test visited links in storybook
+ *
+ * export const Visited = (): React.ReactElement => (
+ *   <p>
+ *     <Link className="usa-color-text-visited" href={'#'}>
+ *       This
+ *     </Link>
+ *     &nbsp;is a visited link.
+ *   </p>
+ * )
+ *  */
+export const DarkBackground = (): React.ReactElement => (
+  <div className="usa-dark-background" style={{ padding: '10px' }}>
+    <p>
+      <Link href={'#'}>This</Link> is a text link on a dark background
+    </p>
+  </div>
+)
+
+export const ExternalLink = (): React.ReactElement => (
+  <p>
+    This is a link that goes to an&nbsp;
+    <Link variant="external" href={'http://www.truss.works'}>
+      external website
+    </Link>
+    .
+  </p>
+)
+
+export const StyledAsButton = (): React.ReactElement => (
+  <p>
+    <Link className="usa-button" variant="unstyled" href={'#'}>
```

Comment:
suggestion for future iteration: I have a feeling the `usa-button` and all of its variations will be a common use-case for `Link` components, so adding the possible button style/className props to the `Link` component might be a good idea.